docs/operators_and_staking/staking.md

```diff
@@ -0,0 +1,128 @@
+
+ ![Staking-7](/img/doc-imgs/operators-staking/Staking-7.png)
+2. Browse the **recent events** and you should see **domains.OperatorNominated** event.
+
+ ![Staking-19](/img/doc-imgs/operators-staking/Staking-19.png)
+3. Click on the dropdown arrow to view the **domainID** and **operatorID**.
+
+---
+
+Alternatively, you can use [Subscan](https://subspace.subscan.io/) which provides a slightly better UI. 
+1. Navigate to **[Subspace Subscan](https://subspace.subscan.io/)** portal.
+2. Click on `Blockchain` -> `Extrinsics`.
+
+  ![Staking-9](/img/doc-imgs/operators-staking/Staking-9.png)
+
+3. Scroll to the bottom of the page to view all recent events, search for `nominate_operator` event.
+
+ ![Staking-20](/img/doc-imgs/operators-staking/Staking-20.png)
+
+4. Click on `Extrinsic ID` for the desired event.
+
+5. Scroll to `Parameters` to view the stake value.
+
+ ![Staking-21](/img/doc-imgs/operators-staking/Staking-21.png)
+
+6. Scroll to `Events` and click on dropdown arrow for `domains(OperatorNominated)`.
+
+  ![Staking-22](/img/doc-imgs/operators-staking/Staking-22.png)
+
+7. Inspect and ensure that `nominatorID` matches your `id`. 
+
+### Stake withdrawal using [Polkadot.js](https://polkadot.js.org/)
+
+Any **operator** or **nominator** can initiate withdrawal. They can withdraw the total staked amount or a portion of their stake.
+
+- If an operator is initiating a withdrawal, then their remaining balance should be at least minimum operator stake, else request is rejected.
+- If a nominator is initiating a withdrawal, and the remaining balance is less than the operator-defined minimum nominator stake, then the total nominator stake is unlocked otherwise, only the requested amount is unlocked.
+
+
+1. Select the account you want to use in `using the selected account`.
+2. Select `domains` under `submit the following extrinsic` and choose `withdrawStake(operatorID, withdraw)` in the dropdown.
+3. Choose an operator by selecting an `operatorID` - in the example, it's set to `17`.
+4. Choose the withdrawal amount in the `withdraw` field - you can specify to withdraw `all` or `some` staking amount. 
+
+:::note
+Example of withdrawal of 1 TSSC stake amount from nominating an operator `17`.
+:::
+
+![Staking-23](/img/doc-imgs/operators-staking/Staking-23.png)
+
+
+Once the withdrawal is submitted, it's finalized after **domain epoch** is completed. All the withdrawn funds are unlocked after the **unlocking period** is complete.
+
+
+### Calculating your nominator balance
+
+1. Proceed to PolkadotJS **[Network Explorer](https://polkadot.js.org/apps/#/explorer)**.
+2. Go to `Developer` -> `Chain state`.
+3. Select `domains` under `selected state query` and `nominators(u64, AccoundID32)`.
+4. Provide the `operatorID` and select your `account` from the dropdown. 
+5. Run the query, remember the `shares` number. 
+  ![Staking-25](/img/doc-imgs/operators-staking/Staking-25.png)
+6. At the same tab, choose `domainStakingSummary(u32)`.
```

Comment:
tab doesn't feel quite right here - dropdown? list? This same problem is on line 121 below. Maybe others too.

Reply:
Hm, list or dropdown sounds a bit unclear in this context, as there are many of them. Maybe "On the same screen" would do better?

Reply:
Happy to take another look at this one but can't seem to find the preview?

Reply:
For this PR I have to manually trigger re-deploy, just created one https://deploy-preview-199--subspace-docs-staging.netlify.app/